R/pdf_document.R

```diff
@@ -109,6 +109,7 @@ pdf_document <- function(toc = FALSE,
   # template path and assets
   if (identical(template, "default")) {
 
+    if (!pandoc_available()) stop("Pandoc not found")
```

Comment:
I think one has to work hard enough to be able to shoot oneself in the foot, since we have set up everything correctly in the IDE, Shiny Server, and RSC... It is possible to print out the places where we searched for pandoc, but I'm not sure if it is worth it.